doc/api/colors_api.rst

```diff
@@ -14,27 +14,44 @@
    :no-members:
    :no-inherited-members:
 
-Classes
--------
+Color norms
+-----------
 
 .. autosummary::
    :toctree: _as_gen/
    :template: autosummary.rst
 
+   NoNorm
+   Normalize
```

Comment:
Maybe `Normalize` first since it's the base class? I assume `NoNorm` is out of order because it's significantly different.

Reply:
Yes, I put NoNorm first, because it's different, but we could put it last as well if you think that's better. I wouldn't want to mix it in between the other norms.

Reply:
I would put `Normalize`, `NoNorm`, then the rest.

Reply:
Also ok by me. Changed to that order.